Google-SPS-Team-5/group-project
# Handle admins/non-admins access to Add New Business form

How admin status works:
We can assign roles to specific accounts on the Google Cloud Platform. Since for all of us, our roles is "owners" of the app "sps Team 43 Summer 20" if we log in with our sps account then we are automatically detected as admins.

To test:
Changes are deployed at [https://summer20-sps-43.appspot.com/](https://summer20-sps-43.appspot.com/). This UserService API does not work on the local dev server.

- Log in with you sps account and log in with your non-sps Google account and see the difference in the navbar. For admins there is an option to click Add New Business. For non-admins, there isn't.
- Go to [https://summer20-sps-43.appspot.com/](https://summer20-sps-43.appspot.com/add.html) as an admin and then as a non-admin. As an admin, you will get access to the Add a New Business Form. For non-admin, a message displays on the webpage and you will be redirected to the homepage after one second.
